plugins/ltac/g_rewrite.mlg

```diff
@@ -278,7 +278,7 @@ VERNAC COMMAND EXTEND AddSetoid1 CLASSIFIED AS SIDEFF
      }
   | #[ atts = rewrite_attributes; ] ![ proof ] [ "Add" "Morphism" constr(m) ":" ident(n) ]
     (* This command may or may not open a goal *)
-    => { VtUnknown, VtNow }
+    => { VtStartProof(GuaranteesOpacity, [n]), VtLater }
```

Comment:
AFAICT this is `if Lib.is_modtype() then sideff else start_proof`, see the add_morphism_infer code.

Reply:
Fixed, thanks!